tests/test_config.py

```diff
@@ -1,7 +1,6 @@
 from pathlib import Path
 
 import pytest
-from deepdiff import DeepDiff
```

Comment:
@briemla we (@randy-seng and me) had to remove this assertion because we couldnt figure out what the exact problem was, but assume it has to do slight inequaliies between some properties of path objects.

But we checked the follwing:

With "==" assertions of two dicts they are checked element-wise for string equality.

As via config we provide str that later will be converted to path, we thought that´s enough to check here.